### PR TITLE
Parse VBMS errors when creating an EP

### PR DESCRIPTION
**Pull request by aroltsch**
connects #3515 

1. Parse all `PersonalInfo` VBMS errors and return the message to the user.
2. If BGS returns a blank zip code, use address line 3 if it contains something resembling a zip code.

### CODE REVIEW

File: app/models/tasks/establish_claim.rb [hunk 1]
```diff
@@ -203,6 +203,8 @@ def parse_vbms_error(error)
       return VBMSError.new("duplicate_ep")
     when /The PersonalInfo SSN must not be empty./
       return VBMSError.new("missing_ssn")
+    when /The PersonalInfo.+must not be empty/
```
Comment (sharonwarner) [inline app/models/tasks/establish_claim.rb]: Wow is that really the error message?? Or is that a typo?

File: app/models/veteran.rb [hunk 2]
```diff
@@ -28,7 +28,7 @@ class Veteran
 
   COUNTRIES_REQUIRING_ZIP = %w(USA CANADA).freeze
 
-  validates :ssn, :first_name, :last_name, :city, :address_line1, :country, presence: true
+  validates :ssn, :sex, :first_name, :last_name, :city, :address_line1, :country, presence: true
```
Comment (sharonwarner) [inline app/models/veteran.rb]: 🎉

File: app/models/tasks/establish_claim.rb (hunk 1 above)
Comment (aroltsch) [reply app/models/tasks/establish_claim.rb]: `.+` is a Ruby regular expression which means "one or more character must be in between strings 'PersonalInfo' and 'must'".

Comment (sharonwarner) [reply app/models/tasks/establish_claim.rb]: Ah... makes sense!

**Pull request merged**